tools/botskills/src/functionality/connectSkill.ts

```diff
@@ -108,8 +108,8 @@ Remember to use the argument '--dispatchFolder' for your Assistant's Dispatch fo
                 }
 
                 if (luFile.trim.length === 0) {
-                    luFile = luFilePath.split('\\').reverse()[0];
-                    luisFile = `${ luFile.toLowerCase() }is`;
+                    luFile = parse(luFilePath).name;
+                    luisFile = luFile + '.luis';
```

Comment:
```suggestion
                    luisFile = `${ luFile }.luis`;
```

For parity with other sections